docs/jdbc-oceanbase-catalog.md

```diff
@@ -0,0 +1,227 @@
+| `jdbc-driver`        | The driver of the JDBC connection. For example, `com.mysql.jdbc.Driver` or `com.mysql.cj.jdbc.Driver`. | (none)        | Yes      | 0.3.0         |
+| `jdbc-user`          | The JDBC user name.                                                                                    | (none)        | Yes      | 0.3.0         |
+| `jdbc-password`      | The JDBC password.                                                                                     | (none)        | Yes      | 0.3.0         |
+| `jdbc.pool.min-size` | The minimum number of connections in the pool. `2` by default.                                         | `2`           | No       | 0.3.0         |
+| `jdbc.pool.max-size` | The maximum number of connections in the pool. `10` by default.                                        | `10`          | No       | 0.3.0         |
+
+:::caution
+You must download the corresponding JDBC driver to the `catalogs/jdbc-oceanbase/libs` directory.
+:::
+
+### Catalog operations
+
+Refer to [Manage Relational Metadata Using Gravitino](./manage-relational-metadata-using-gravitino.md#catalog-operations) for more details.
+
+## Schema
+
+### Schema capabilities
+
+- Gravitino's schema concept corresponds to the OceanBase database.
+- Supports creating schema, but does not support setting comment.
+- Supports dropping schema.
+- Supports cascade dropping schema.
+
+### Schema properties
+
+- Doesn't support any schema property settings.
+
+### Schema operations
+
+Refer to [Manage Relational Metadata Using Gravitino](./manage-relational-metadata-using-gravitino.md#schema-operations) for more details.
+
+## Table
+
+### Table capabilities
+
+- Gravitino's table concept corresponds to the OceanBase table.
+- Supports DDL operation for OceanBase tables.
+- Supports index.
+- Supports [column default value](./manage-relational-metadata-using-gravitino.md#table-column-default-value) and [auto-increment](./manage-relational-metadata-using-gravitino.md#table-column-auto-increment)..
+
+#### Table column types
+
+| Gravitino Type    | OceanBase Type      |
+|-------------------|---------------------|
+| `Byte`            | `Tinyint`           |
+| `Byte(false)`     | `Tinyint Unsigned`  |
+| `Short`           | `Smallint`          |
+| `Short(false)`    | `Smallint Unsigned` |
+| `Integer`         | `Int`               |
+| `Integer(false)`  | `Int Unsigned`      |
+| `Long`            | `Bigint`            |
+| `Long(false)`     | `Bigint Unsigned`   | 
+| `Float`           | `Float`             |
+| `Double`          | `Double`            |
+| `String`          | `Text`              |
+| `Date`            | `Date`              |
+| `Time`            | `Time`              |
+| `Timestamp`       | `Timestamp`         |
+| `Decimal`         | `Decimal`           |
+| `Decimal`         | `Numberic`          |
```

Comment:
When using `Decimal` in Gravitino to create an OB table, which OB type will be chosen?

Reply:
Fixed, `Decimal` will be chosen.